src/astro/utils/snowflake_transform.py

```diff
@@ -20,7 +20,11 @@
 
 
 def handle_table(t: Table):
-    return t.schema + "." + t.table_name if t.schema else t.table_name
+    return (
+        t.database + "." + t.schema + "." + t.table_name
+        if t.schema and "." not in t.table_name
+        else t.table_name
```

Comment:
In which circumstances do we expect `table_name` to contain  `.`?

Reply:
@tatiana if the user provides a full tablename path. I've seen multiple test users in snowflake choose to identify the entire table name "DATABASE"."SCHEMA"."TABLE NAME"

Reply:
@tatiana I added a docstring and made it more explicit

Reply:
Actually let's just always do fully qualified name. I'm worried there will be edge cases with table names that have period in them.